src/components/NavBar/connecttowallet.tsx

```diff
@@ -0,0 +1,21 @@
+import React, {useState} from 'react';
```

Comment:
the name of the file should be in UpperCamelCase style
https://betterprogramming.pub/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841